common/src/main/resources/assets/gtceu/compass/pages/en_us/materials/purified_ore.xml

```diff
@@ -0,0 +1,17 @@
+<page>
+  <h1>Purified Ore</h1>
+  <h3>Source</h3>
+  <text>
+    <style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki. <br></br>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/119280/">Oganesson897</style>, <br></br> <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://github.com/blackdragon2447">BlackDragon2447</style> <br></br></style>
+  </text>
+  <text>
+    <style color="0xaaaaaa">(Purified ore is an intermediary product in the processing of ores, see the ore processing tab in JEI/REI/EMI for any ore's specific processing steps)</style>
+  </text>
+  <br></br>
+  <text>Crushed ore is obtained by processing <style color="0xafff00" hover-info="click link" link="gtceu:materials/crushed_ore" underlined="true">Crushed Ore</style> in a <style color="0xafff00" hover-info="click link" link="gtceu:machines/ore_wahser" underlined="true">Ore Washer</style> or, for some ores, a <style color="0xafff00" hover-info="click link" link="gtceu:machines/chemical_bath" underlined="true">Chemical Bath</style>, where both processes yield diffiernt byproducts. Purified ore can be further processed into <style color="0xafff00" hover-info="click link" link="gtceu:materials/purified_dust" underlined="true">Purified Dust</style> using a <style color="0xafff00" hover-info="click link" link="gtceu:machines/macerator" underlined="true">Macerator</style> or <style color="0xafff00" hover-info="click link" link="gtceu:machines/forge_hammer" underlined="true">Forge Hammer</style> or into <style color="0xafff00" hover-info="click link" link="gtceu:materials/refined_ore" underlined="true">Refined Ore</style> in a <style color="0xafff00" hover-info="click link" link="gtceu:machines/thermal_centrifuge" underlined="true">Thermal Centrifuge</style>.</text>
```

Comment:
diffiernt

Reply:
Guess that one slipped through the spell checker.